cmd/operator-sdk/scorecard/cmd.go

```diff
@@ -15,26 +15,40 @@
 package scorecard
 
 import (
+	"bytes"
 	"fmt"
 	"log"
+	"os"
 
+	"github.com/mitchellh/mapstructure"
 	"github.com/operator-framework/operator-sdk/internal/scorecard"
 	schelpers "github.com/operator-framework/operator-sdk/internal/scorecard/helpers"
 	scplugins "github.com/operator-framework/operator-sdk/internal/scorecard/plugins"
+	"github.com/operator-framework/operator-sdk/internal/util/projutil"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/labels"
 
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
```

Comment:
The imports order is not following the standard. Following the suggestion. 

```suggestion
	"bytes"
	"fmt"
	"log"
	"os"

	"github.com/mitchellh/mapstructure"
	"github.com/pkg/errors"
	"k8s.io/apimachinery/pkg/labels"
	"github.com/spf13/cobra"
	"github.com/spf13/viper"
	
	"github.com/operator-framework/operator-sdk/internal/scorecard"
	schelpers "github.com/operator-framework/operator-sdk/internal/scorecard/helpers"
	scplugins "github.com/operator-framework/operator-sdk/internal/scorecard/plugins"
	"github.com/operator-framework/operator-sdk/internal/util/projutil"
```

Reply:
good catch, should be ordered according to goimports formatting in next commit.